classes/class4/homework-soln/routes/index.js

```diff
@@ -0,0 +1,165 @@
+/* CONNECT TO MONGOOSE */
+var mongoose = require('mongoose');
+mongoose.connect(process.env.MONGOURI || 'mongodb://localhost/test');
+
+
+/* DEFINE SCHEMAS */
+var ingredientSchema = mongoose.Schema({
```

Comment:
Schemas and models would ideally be specified and then exported from a new file in a /models folder.